fvm/src/kernel/mod.rs

```diff
@@ -341,19 +307,15 @@ pub trait EventOps {
     ) -> Result<()>;
 }
 
-// Unfortunately, I need to do this to make it possible to name these macros by path. I'm hiding
-// this because I really don't want users to glob import the `kernel` module.
-#[doc(hidden)]
-pub use {
```

Comment:
automatically exported now.

Reply:
do we have a breaking change now though because these traits are no longer exported on the crate root? (as per [notes](https://docs.rs/ambassador/0.4.0/ambassador/#03x---04x)); or do we consider this unlikely to really impact anyone?

Reply:
Ah, you're right. I'll make a note of it but I doubt it'll impact anyone.